CHANGELOG.md

```diff
@@ -10,7 +10,9 @@ _None_
 
 ### New Features
 
-_None_
+* Add SwiftPM `Bundle.module` compatibility to all templates.  
```

Comment:
```suggestion
* The built-in templates will now correctly handle Swift Package Manager resources, using `Bundle.module` if it's available. As before, you can override the used `Bundle` using the `bundle` or `lookupFunction` template parameters.  
```

Not 100% sure this is the right terminology in the 1st sentence 🤷

Reply:
This is better thanks 👍 

We could test each source with a SwiftPM 5.3 bundle but, doing this, I guess it will be resumed to test if SwiftPM is working and not `SwiftGen`? 🤔

Reply:
Hmmm what we could do is test the output once without that flag defined, and once with SWIFT_PACKAGE defined, to ensure both versions compile. I think that involves a slight refactor to the test yaml files though 🤔 Stuff for another PR for sure.

Reply:
OK but `Bundle.module` is generated only when `Package.swift` has defined `resources`.
So even if `SWIFT_PACKAGE` is defined we may have a compiler error when `Package.swift` has no `resources` (it make sense because the bundle can not exist without resources).

So to test it we will have to create a package with defined resources first to generate the `Bundle.module` boilerplate code.

Reply:
No need for that actually, we can perfectly fake it like we do in other tests, just include a bit of code that defines a static `let` on `Bundle` called `module`. That part is easy, and we support it already.

The bit that'll be a bit more involved, is allowing a file to be compiled multiple times with different configurations. A compilation configuration like this one:
https://github.com/SwiftGen/SwiftGen/blob/stable/Tests/Fixtures/Generated/IB-iOS/scenes-swift5/compilation-configuration.yml
Will need to be changed somehow to allow variations per file (the keys in `file_specific`).

Like I said, issue for a follow-up ticket/PR.